maps/RoidStation.dmm

```diff
@@ -3799,7 +3799,7 @@
 	},
 /obj/machinery/atmospherics/pipe/layer_adapter/scrubbers/hidden,
 /turf/simulated/floor/vox/wood,
-/area/vox_trading_post/dorms)
+/area/crew_quarters/sleep)
```

Comment:
you can't just use this area on the vox post, this means it will share an APC/power with the dorms on the main station

Reply:
Oh, I thought they needed to be connected. Dreamseeker didn't warn me about double APCs.

I've fixed this by putting the vox_trading_post and child area's in immune area's in code. For some reason, we have both a vox_station and vox_trading_post parent area.

Reply:
Yes, dreamseeker won't warn you, it will just delete one APC at runtime. Which means either the Vox will get an APC that can control the station APC or visa versa.

That's fine.